IntervalTree.h

```diff
@@ -61,7 +61,7 @@ class IntervalTree {
     IntervalTree()
         : left(nullptr)
         , right(nullptr)
-        , center(0)
+        , center(Scalar{})
```

Comment:
nit: if `center` is already a `Scalar`, you can use `center()`. Otherwise you _may_ invoke the copy constructor (!) or move constructor, which you may not want.

Reply:
![20191016_140003.jpg](https://user-images.githubusercontent.com/121945228/211972061-5ffd4c3e-82a7-4b96-b8a3-c0753d8013ea.jpg)